pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -568,13 +570,15 @@ protected void setupCustomRouting(@NonNull Config cfg) {
         get(apiRoot + RoutePath.BSP_KIT_QUERY_PATH, new BSPKitRoute(notificationUtil), new JsonTransformer());
         get(apiRoot + RoutePath.BSP_KIT_REGISTERED, new BSPKitRegisteredRoute(), new JsonTransformer());
         get(apiRoot + RoutePath.CLINICAL_KIT_ENDPOINT, new ClinicalKitsRoute(notificationUtil), new JsonTransformer());
-        get(apiRoot + RoutePath.CREATE_CLINICAL_KIT_ENDPOINT, new CreateClinicalDummyKitRoute(new OncHistoryDetailDaoImpl()),
-                new JsonTransformer());
-        get(apiRoot + RoutePath.CREATE_CLINICAL_KIT_ENDPOINT_WITH_PARTICIPANT,
-                new CreateClinicalDummyKitRoute(new OncHistoryDetailDaoImpl()),
-                new JsonTransformer());
+
+        NotPepperKitCreationService notPepperKitCreationService = new NotPepperKitCreationService();
+        get(apiRoot + RoutePath.SHIP_KIT_ENDPOINT, new JuniperShipKitRoute(notPepperKitCreationService), new JsonTransformer());
```

Comment:
New Route for kit creation

Reply:
I thought more about this an have an opinion, FWIW. 
The thing that's bothering me are the terms 'notPepper' and 'nonPepper' since they don't fit with the rest of the DSM code. IOW, we don't use the term Pepper in the code (at least not in any prominent places), so calling something non-pepper does not fit with the code.
Could we change 'notPepper' to 'external' throughout this code, and just leave the references to 'Juniper' as is? Does that make sense?

Reply:
external in terms of DSM vocabulary has a different meaning, and that is when a kit is shipped through or working in a non-broad GP

I am more than happy to change the names to Juniper and keep thinking about a good name. Since it is a first draft, I think we can push this to dev so Juniper team can use it while we come up with a better name. To me, for now, Juniper will do, especially since I don't think we will make this service for anything else. What do you think?

We can also create a ticket to come up with a better name, put the code in dev so Juniper engineers can use it and change the name later